hil/ext/switches/brocade.py

```diff
@@ -100,10 +116,14 @@ def _enable_and_set_mode(self, interface, mode):
             mode: 'access' or 'trunk'
 
         Raises: AssertionError if mode is invalid.
-
         """
-        # Enable switching
         url = self._construct_url(interface)
+
+        # Turn on port; equivalent to running `no shutdown` on the switchport
+        if self._is_interface_off:
```

Comment:
mistake here. didn't call the method correctly and this was calling the deployment tests to fail.